manager/controllers/package_systems.go

```diff
@@ -25,7 +25,7 @@ func packageSystemsQuery(acc string, pkgName string) *gorm.DB {
 	return database.Db.
 		Table("system_platform").
 		Joins("inner join rh_account ra on system_platform.rh_account_id = ra.id").
-		Joins("inner join system_package spkg on spkg.system_id = system_platform.id").
+		Joins("inner join system_package spkg on  on spkg.rh_account_id = ra.id AND spkg.system_id = system_platform.id").
```

Comment:
on  on spkg.rh_account_id = ra.id AND 
^^ duplicated keyword

Reply:
Fixed

Reply:
I'd prefer to fix it in this commit not next one, just for the sake of clarity. ;)